finality-aleph/src/nodes/validator_node.rs

```diff
@@ -42,10 +48,40 @@ where
         millisecs_per_block,
         justification_rx,
         backup_saving_path,
+        external_addresses,
+        validator_port,
         ..
     } = aleph_config;
 
-    let (validator_network, validator_network_identity) = tcp_network::new_noop().await;
+    // We generate the phrase manually to only save the key in RAM.
+    let peer_id = keystore
+        .ed25519_generate_new(
+            KEY_TYPE,
```

Comment:
If this is the network key, then we should definitely use a different `KEY_TYPE`. The current one is dedicated for aleph-bft consensus.

Reply:
Created a specific one for the validator network and used it here.